docs/intro/index.rst

```diff
@@ -4,32 +4,33 @@
 Introduction
 ============
 
-**dwave-hybrid** provides a framework for iterating arbitrary-sized sets of samples
-through parallel solvers to find an optimal solution.
+**dwave-hybrid** provides a framework for iterating arbitrary-sized sets of 
+samples through parallel solvers to find an optimal solution.
 
-For the documentation of a particular code element, see the :ref:`reference_hybrid` section.
-This introduction gives an overview of the package; steps you through using it,
-starting with running a provided hybrid solver that handles arbitrary-sized QUBOs; and
-points out the way to developing your own components in the framework.
+For the documentation of a particular code element, see the 
+:ref:`reference_hybrid` section. This introduction gives an overview of the 
+package; steps you through using it, starting with running a provided hybrid
+solver that handles arbitrary-sized QUBOs; and points out the way to developing 
+your own components in the framework.
 
-* :ref:`overview_hybrid` presents the framework and explains key concepts.
+*   :ref:`overview_hybrid` presents the framework and explains key concepts.
 
-* :ref:`using_framework` shows how to use the framework. You can quickly get started by using a
-  provided reference sampler built with this framework, Kerberos, to solve a problem too
-  large to :term:`minor-embed` on a D-Wave system. Next, use the framework
-  to build (hybrid) workflows; for example, a solver similar to :std:doc:`qbsolv <oceandocs:docs_qbsolv>`,
-  which can employ tabu search on a whole problem while submitting parts of the problem
-  to a D-Wave system.
+*   :ref:`using_framework` shows how to use the framework. You can quickly get 
+    started by using a provided reference sampler built with this framework, 
+    :class:`Kerberos <oceandocs:hybrid.reference.kerberos.KerberosSampler>`, to 
+    solve a problem too large to :term:`minor-embed` on a D-Wave system. Next, 
+    use the framework to build (hybrid) workflows; for example, a workflow for 
+    larger-than-QPU lattice-structured problems.
 
-* :ref:`developing_hybrid` guides you to developing your own hybrid components.
+*   :ref:`developing_hybrid` guides you to developing your own hybrid components.
 
-* :ref:`reference_examples_hybrid` describes some workflow examples included in the code.
+*   :ref:`reference_examples_hybrid` describes some workflow examples included in the code.
 
 .. toctree::
-   :maxdepth: 2
-   :hidden:
+    :maxdepth: 2
+    :hidden:
 
-   overview
-   using
-   developing
-   reference_examples
+    overview
+    using
+    developing
+    reference_examples
```

Comment:
Frivolous indentation changes are best coupled with bigger semantic changes to not pollute/add noise to git history. Alternatively, one can separate formatting changes in a separate commit.

Reply:
I regret not fixing the indentation in a separate commit. I deny the charge of friviolity.